src/code/utils/undo-redo.ts

```diff
@@ -49,9 +50,10 @@ export class UndoRedoManager {
     this.savePosition = -1;
     this.changeListeners = [];
     this.currentBatch = null;
+    this.commandBatchingEnabled = true;
 
     // listen to all our actions
-    this.endCommandBatch.listen(this._endComandBatch, this);
+    this.endCommandBatch.listen(this._endCommandBatch, this);
```

Comment:
Errors like this typo make me wonder why/how software works at all.  Nice catch, and how did things ever work before?

Reply:
The `_endComandBatch` was a private method not called outside the class wrapped by a public `endCommandBatch` method so it did all work before, but I thought since I saw it it would be good to fix.